exercises/connect/.meta/template.j2

```diff
@@ -0,0 +1,10 @@
+{%- extends "master_template.j2" -%}
+{% set imports = ["ConnectGame"] %}
+
+{% macro test_case(case) -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        game = ConnectGame("""{{ case["input"]["board"] | join("\n               ") }}""")
```

Comment:
Unless @yawpitch knows a more elegant way to generate these boards, LGTM.

Reply:
Nothing I can think of off the top of my head that will do it better.